### ISSUE TITLE
neither `tidy` nor `glance` methods work for `gee` objects

### ISSUE THREAD

**Issue by IndrajeetPatil** [tidymodels/broom#708]
Objects of class `gee` are not supported but because it also inherits `glm` class, I guess that's the tidier that gets picked up.

``` r
# setup
set.seed(123)
library(MASS)
library(gee)
data(OME)

# model
fm <-
  gee(
    cbind(Correct, Trials - Correct) ~ Loud + Age + OME,
    id = ID,
    data = OME,
    family = binomial,
    corstr = "exchangeable"
  )
#> Beginning Cgee S-function, @(#) geeformula.q 4.13 98/01/27
#> running glm to get initial regression estimate
#> (Intercept)        Loud         Age     OMEhigh      OMElow 
#> -5.90984676  0.15516993  0.01876291 -0.07569691 -0.31660542

# class of object
class(fm)
#> [1] "gee" "glm"

# summary
summary(fm)
#> 
#>  GEE:  GENERALIZED LINEAR MODELS FOR DEPENDENT DATA
#>  gee S-function, version 4.13 modified 98/01/27 (1998) 
#> 
#> Model:
#>  Link:                      Logit 
#>  Variance to Mean Relation: Binomial 
#>  Correlation Structure:     Exchangeable 
#> 
#> Call:
#> gee(formula = cbind(Correct, Trials - Correct) ~ Loud + Age + 
#>     OME, id = ID, data = OME, family = binomial, corstr = "exchangeable")
#> 
#> Summary of Residuals:
#>        Min         1Q     Median         3Q        Max 
#> -0.9585811  0.5507230  2.0414189  3.1210663 12.0241951 
#> 
#> 
#> Coefficients:
#>                Estimate  Naive S.E.     Naive z Robust S.E.    Robust z
#> (Intercept) -5.90100336 0.336455218 -17.5387482 0.231463368 -25.4943295
#> Loud         0.15507565 0.007513314  20.6401142 0.005321880  29.1392592
#> Age          0.01851319 0.003561640   5.1979385 0.003305525   5.6006793
#> OMEhigh     -0.04183516 0.160288285  -0.2609995 0.152182352  -0.2749015
#> OMElow      -0.28563252 0.131496124  -2.1721745 0.117510247  -2.4307031
#> 
#> Estimated Scale Parameter:  1.250466
#> Number of Iterations:  4
#> 
#> Working Correlation
#>              [,1]        [,2]        [,3]        [,4]        [,5]
#>  [1,]  1.00000000 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [2,] -0.02344803  1.00000000 -0.02344803 -0.02344803 -0.02344803
#>  [3,] -0.02344803 -0.02344803  1.00000000 -0.02344803 -0.02344803
#>  [4,] -0.02344803 -0.02344803 -0.02344803  1.00000000 -0.02344803
#>  [5,] -0.02344803 -0.02344803 -0.02344803 -0.02344803  1.00000000
#>  [6,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [7,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [8,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [9,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [10,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [11,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [12,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [13,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [14,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [15,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [16,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [17,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [18,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [19,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [20,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [21,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [22,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [23,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [24,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [25,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [26,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [27,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [28,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [29,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [30,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>              [,6]        [,7]        [,8]        [,9]       [,10]
#>  [1,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [2,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [3,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [4,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [5,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [6,]  1.00000000 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [7,] -0.02344803  1.00000000 -0.02344803 -0.02344803 -0.02344803
#>  [8,] -0.02344803 -0.02344803  1.00000000 -0.02344803 -0.02344803
#>  [9,] -0.02344803 -0.02344803 -0.02344803  1.00000000 -0.02344803
#> [10,] -0.02344803 -0.02344803 -0.02344803 -0.02344803  1.00000000
#> [11,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [12,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [13,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [14,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [15,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [16,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [17,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [18,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [19,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [20,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [21,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [22,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [23,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [24,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [25,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [26,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [27,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [28,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [29,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [30,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>             [,11]       [,12]       [,13]       [,14]       [,15]
#>  [1,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [2,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [3,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [4,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [5,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [6,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [7,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [8,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [9,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [10,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [11,]  1.00000000 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [12,] -0.02344803  1.00000000 -0.02344803 -0.02344803 -0.02344803
#> [13,] -0.02344803 -0.02344803  1.00000000 -0.02344803 -0.02344803
#> [14,] -0.02344803 -0.02344803 -0.02344803  1.00000000 -0.02344803
#> [15,] -0.02344803 -0.02344803 -0.02344803 -0.02344803  1.00000000
#> [16,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [17,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [18,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [19,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [20,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [21,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [22,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [23,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [24,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [25,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [26,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [27,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [28,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [29,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [30,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>             [,16]       [,17]       [,18]       [,19]       [,20]
#>  [1,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [2,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [3,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [4,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [5,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [6,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [7,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [8,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [9,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [10,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [11,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [12,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [13,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [14,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [15,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [16,]  1.00000000 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [17,] -0.02344803  1.00000000 -0.02344803 -0.02344803 -0.02344803
#> [18,] -0.02344803 -0.02344803  1.00000000 -0.02344803 -0.02344803
#> [19,] -0.02344803 -0.02344803 -0.02344803  1.00000000 -0.02344803
#> [20,] -0.02344803 -0.02344803 -0.02344803 -0.02344803  1.00000000
#> [21,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [22,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [23,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [24,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [25,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [26,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [27,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [28,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [29,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [30,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>             [,21]       [,22]       [,23]       [,24]       [,25]
#>  [1,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [2,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [3,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [4,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [5,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [6,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [7,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [8,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [9,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [10,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [11,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [12,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [13,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [14,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [15,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [16,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [17,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [18,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [19,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [20,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [21,]  1.00000000 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [22,] -0.02344803  1.00000000 -0.02344803 -0.02344803 -0.02344803
#> [23,] -0.02344803 -0.02344803  1.00000000 -0.02344803 -0.02344803
#> [24,] -0.02344803 -0.02344803 -0.02344803  1.00000000 -0.02344803
#> [25,] -0.02344803 -0.02344803 -0.02344803 -0.02344803  1.00000000
#> [26,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [27,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [28,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [29,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [30,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>             [,26]       [,27]       [,28]       [,29]       [,30]
#>  [1,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [2,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [3,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [4,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [5,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [6,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [7,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [8,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#>  [9,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [10,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [11,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [12,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [13,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [14,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [15,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [16,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [17,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [18,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [19,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [20,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [21,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [22,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [23,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [24,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [25,] -0.02344803 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [26,]  1.00000000 -0.02344803 -0.02344803 -0.02344803 -0.02344803
#> [27,] -0.02344803  1.00000000 -0.02344803 -0.02344803 -0.02344803
#> [28,] -0.02344803 -0.02344803  1.00000000 -0.02344803 -0.02344803
#> [29,] -0.02344803 -0.02344803 -0.02344803  1.00000000 -0.02344803
#> [30,] -0.02344803 -0.02344803 -0.02344803 -0.02344803  1.00000000

# tidy output
broom::tidy(fm)
#> Column 6 must be named.
#> Use .name_repair to specify repair.

# glance
broom::glance(fm)
#> All columns in a tibble must be 1d or 2d objects:
#> * Column `null.deviance` is NULL
#> * Column `df.null` is NULL
#> * Column `deviance` is NULL
#> * Column `df.residual` is NULL
```

<sup>Created on 2019-05-28 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

<details>

<summary>Session info</summary>

``` r
devtools::session_info()
#> - Session info ----------------------------------------------------------
#>  setting  value                                    
#>  version  R version 3.6.0 alpha (2019-03-29 r76300)
#>  os       Windows 10 x64                           
#>  system   x86_64, mingw32                          
#>  ui       RTerm                                    
#>  language (EN)                                     
#>  collate  English_United States.1252               
#>  ctype    English_United States.1252               
#>  tz       America/New_York                         
#>  date     2019-05-28                               
#> 
#> - Packages --------------------------------------------------------------
#>  package     * version    date       lib source                         
#>  assertthat    0.2.1      2019-03-21 [1] CRAN (R 3.6.0)                 
#>  backports     1.1.4      2019-04-10 [1] CRAN (R 3.6.0)                 
#>  broom         0.5.2.9001 2019-04-09 [1] local                          
#>  callr         3.2.0      2019-03-15 [1] CRAN (R 3.6.0)                 
#>  cli           1.1.0      2019-03-19 [1] CRAN (R 3.6.0)                 
#>  crayon        1.3.4      2017-09-16 [1] CRAN (R 3.5.1)                 
#>  desc          1.2.0      2019-04-03 [1] Github (r-lib/desc@c860e7b)    
#>  devtools      2.0.2      2019-04-08 [1] CRAN (R 3.6.0)                 
#>  digest        0.6.19     2019-05-20 [1] CRAN (R 3.6.0)                 
#>  dplyr         0.8.1      2019-05-14 [1] CRAN (R 3.6.0)                 
#>  evaluate      0.14       2019-05-28 [1] CRAN (R 3.6.0)                 
#>  fs            1.3.1      2019-05-06 [1] CRAN (R 3.6.0)                 
#>  gee         * 4.13-19    2015-06-29 [1] CRAN (R 3.6.0)                 
#>  generics      0.0.2      2019-03-05 [1] Github (r-lib/generics@c15ac43)
#>  glue          1.3.1      2019-03-12 [1] CRAN (R 3.6.0)                 
#>  highr         0.8        2019-03-20 [1] CRAN (R 3.6.0)                 
#>  htmltools     0.3.6      2017-04-28 [1] CRAN (R 3.5.1)                 
#>  knitr         1.23       2019-05-18 [1] CRAN (R 3.6.0)                 
#>  magrittr      1.5        2014-11-22 [1] CRAN (R 3.5.1)                 
#>  MASS        * 7.3-51.4   2019-03-31 [1] CRAN (R 3.6.0)                 
#>  memoise       1.1.0      2017-04-21 [1] CRAN (R 3.6.0)                 
#>  pillar        1.4.1      2019-05-28 [1] CRAN (R 3.6.0)                 
#>  pkgbuild      1.0.3      2019-03-20 [1] CRAN (R 3.6.0)                 
#>  pkgconfig     2.0.2      2018-08-16 [1] CRAN (R 3.5.1)                 
#>  pkgload       1.0.2      2018-10-29 [1] CRAN (R 3.6.0)                 
#>  prettyunits   1.0.2      2015-07-13 [1] CRAN (R 3.5.1)                 
#>  processx      3.3.1      2019-05-08 [1] CRAN (R 3.6.0)                 
#>  ps            1.3.0      2018-12-21 [1] CRAN (R 3.6.0)                 
#>  purrr         0.3.2      2019-03-15 [1] CRAN (R 3.6.0)                 
#>  R6            2.4.0      2019-02-14 [1] CRAN (R 3.6.0)                 
#>  Rcpp          1.0.1      2019-03-17 [1] CRAN (R 3.6.0)                 
#>  remotes       2.0.4      2019-04-10 [1] CRAN (R 3.6.0)                 
#>  rlang         0.3.4      2019-04-07 [1] CRAN (R 3.6.0)                 
#>  rmarkdown     1.13       2019-05-22 [1] CRAN (R 3.6.0)                 
#>  rprojroot     1.3-2      2018-01-03 [1] CRAN (R 3.5.1)                 
#>  sessioninfo   1.1.1      2018-11-05 [1] CRAN (R 3.6.0)                 
#>  stringi       1.4.3      2019-03-12 [1] CRAN (R 3.6.0)                 
#>  stringr       1.4.0      2019-02-10 [1] CRAN (R 3.6.0)                 
#>  testthat      2.1.1      2019-04-23 [1] CRAN (R 3.6.0)                 
#>  tibble        2.1.1      2019-03-16 [1] CRAN (R 3.6.0)                 
#>  tidyr         0.8.3      2019-03-01 [1] CRAN (R 3.6.0)                 
#>  tidyselect    0.2.5      2018-10-11 [1] CRAN (R 3.5.1)                 
#>  usethis       1.5.0      2019-04-07 [1] CRAN (R 3.6.0)                 
#>  withr         2.1.2      2018-03-15 [1] CRAN (R 3.5.1)                 
#>  xfun          0.7        2019-05-14 [1] CRAN (R 3.6.0)                 
#>  yaml          2.2.0      2018-07-25 [1] CRAN (R 3.5.1)                 
#> 
#> [1] C:/Users/inp099/Documents/R/win-library/3.6
#> [2] C:/Program Files/R/R-3.6.0alpha/library
```

</details>

**Comment by alexpghayes** [comment]
The short term solution is:

```r
#' @export
tidy.gee <- tidy.default
```

and analogous methods for `glance()` and `augment()`. The long term solution is proper tidiers.

**Comment by alexpghayes** [comment]
@jyuu This could be a six liner or you could write new tidiers from scratch

**Comment by alexpghayes** [comment]
Closed in #742
